source/sync/configure/sync-settings.txt

```diff
@@ -0,0 +1,618 @@
+
+Performance and Storage
+```````````````````````
+
+Queryable fields use storage on the backing Atlas cluster. The more queryable
+fields you configure, the more storage you use on the backing cluster. You can
+configure up to 10 queryable fields per application for Flexible Sync.
+
+.. seealso::
+
+   Learn more about :ref:`optimizing performance and storage when using Flexible Sync <optimizing-performance-and-storage-flexible-sync>`.
+
+Consequences of Adding or Removing Queryable Fields
+```````````````````````````````````````````````````
+
+You can :ref:`update your Sync configuration <alter-your-sync-configuration>` to
+add or remove queryable field names, but be aware of the following:
+
+When you add a queryable field, devices can only sync on that field once the
+device has caught up to the point in time in :ref:`Device Sync History
+<sync-history>` where the field was added. 
+
+When you remove a queryable field, any devices still using that field will have
+their Device Sync session dropped and receive a :ref:`client reset
+<client-resets>`. Clients not using the removed field won't receive any errors.
+To avoid triggering a client reset when you remove the queryable field, you
+should first remove usage of that field on the client-side.
+
+
+Permissions
+~~~~~~~~~~~
+
+Refer to :ref:`sync-permissions`.
+
+.. _asymmetric-sync:
+.. _optimize-asymmetric-sync:
+
+Asymmetric Sync
+~~~~~~~~~~~~~~~
+
+Asymmetric Sync is a sync strategy for applications with heavy client-side
+insert-only workloads. You can enable it for one or more collections. It
+supports writing to any collection type, including an Atlas time-series
+collection.
+
+For example, an IoT app that frequently logs sensor data has a significant write
+workload and no read workload. The device may also be offline for extended
+periods of time. Asymmetric Sync bypasses some of the processing required for
+bi-directional sync, significantly improving write speed to an Atlas collection.
+
+Other use cases include writing immutable data, such as invoices from a retail
+app, or logging application events, neither of which requires conflict
+resolution.
+
+You can apply Asymmetric Sync to individual collections. This means your app can
+use Asymmetric Sync to write some data, but bi-directional Flexible Sync on
+other collections.
+
+Asymmetric Sync collections are only for writing data. You cannot use Flexible
+Sync queries against these collections. Instead, use :ref:`data-sources`.
+
+After you have enabled Asymmetric Sync, you implement it in the client app via
+the client SDKs. Currently, only the following SDKs support Asymmetric Sync:
+
+- JavaScript
+- React Native
+
+Client Max Offline Time
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Refer to :ref:`client-maximum-offline-time`.
+
+Client Recovery
+~~~~~~~~~~~~~~~
+
+Refer to :ref:`recover-unsynced-changes`.
+
+.. _appconfig-sync:
+.. _sync-configuration-reference:
+
+Sync Configuration File Reference
```

Comment:
Love having this here ❤️  Hopefully this means we will not forget to update it when settings change!